
# HG changeset patch
# User Stuart Rackham <srackham@methods.co.nz>
# Date 1196022383 -46800
# Node ID 7c499794d1cb583bdbe3303c46ee7274070d2041
# Parent fd93b0788f7ddec6abf9dbb4cb24b31a6afcb694
added DocBook refentry lang attribute -- patch contributed by VMiklos

--- a/docbook.conf	Thu Nov 22 09:33:14 2007 +1300
+++ b/docbook.conf	Mon Nov 26 09:26:23 2007 +1300
@@ -424,7 +424,7 @@ ifdef::doctype-manpage[]
 
 [header]
 template::[header-declarations]
-<refentry>
+<refentry lang="{lang=en}">
 <refmeta>
 <refentrytitle>{mantitle}</refentrytitle>
 <manvolnum>{manvolnum}</manvolnum>

